packages/fxa-auth-client/test/crypto.ts

```diff
@@ -36,4 +40,28 @@ describe('lib/crypto', () => {
       );
     });
   });
+  
+  describe('jwe', () => {
+   it('should encrypt and decrypt',  async () => {
+      const recoveryKey = await randomKey();
+      const encoder = new TextEncoder();
+      const salt = hexToUint8(uid);
+      const data = {
+       kB: "aaaa"
+      };
+     const recoveryKeyId = await getRecoveryKeyIdByUid(recoveryKey, uid);
```

Comment:
Indentation feels 1-3 spaces off in this test/suite. But I think the git hooks should be running Prettier against .ts|tsx files.

https://github.com/mozilla/fxa/blob/67d16b354035b306177120816405295c74b1ca2d/package.json#L64-L66